### DISCUSSION TITLE
DHCP not working on eth0 after setting up

### DISCUSSION THREAD

**Discussion by fabsenet** [RaspAP/raspap-webgui#1037] [Q&A]
Hi,

i want to manage to connect to a single wired device over a wireless connection. Schema:

`notebook )))    (((  PI ---- wiredDevice`

there is no internet access involved at all in this setup. I want my notebook directly talking to the wireddevice.

I managed to get raspap configured for wifi and dhcp and everything works for my notebook. I set up eth0 with a static address and enabled all dhcp settings with the same data as for wlan0 except making it another network by using 10.**4**.141. ...

Problem is now: my wiredDevice does not get a IP adress at all. DHCP logging in raspap does not list anything.

I was reading #324 and it should work, shouldn't it? Please help me, as I do not really know where to even start here

**Comment by billz** [comment]
Best advice I can offer is to follow the steps outlined here https://github.com/RaspAP/raspap-webgui/issues/324#issuecomment-725995516

Re: logging, be sure to enable the **Log DHCP requests** option in **DHCP > Logging**, then restart dnsmasq. The service must be restarted in order for logging to work. In most cases log output will help you identify the source of the problem.

See also https://docs.raspap.com/ap-basics/#troubleshooting
